src/examples/regression_test.json

```diff
@@ -24,7 +24,7 @@
+      "data": "AACYleKdZbT/a5vtaOprNqqTIsYqzmPEkcrZ+6OsalhEB78rwASNIdHL4GB76R4W2YNnSxj3tJf8fRy+6MZ4Zl0vRsTGgMVzYsH3VjaSv30IiYmsFGrKkDHixTxoEqVlpxQ46WPwaYWEo9SWRXpuaDKVbbZRD/1KiyOifIoEKbo1JFG8xrNqSAPmpYv0peN5nmGV/VXWW7HZXWQpu5Kh+586dYTsT+LW08yA5tndb2TYz3qDyqyPnvOpTRdgVFmldSX7JzQdPxr5O1P26rJvOkC7NDWLAQYP02gEVu312pHuGjm4Ra3Mjap0NSIw/cCGroJCdPB3Wq2oEpy/urMBHuQBK2P4XpBpuqt1hhkBS2jSGwz47FnMeaKvYaLgmY//0ix/f67LshCF4lXbLxzBLtTgaFnG/xHSvTIdKnN3x66TJeyU+8gvUQ4b3xrKxNgRmaS/cxyZ6tfOaoPWeTW9sgE2WzBw7eKTzzMb09cJWw7KSzq93Zkx12JJN+cA2MF0DQxum82Ai4ELYbM2s5mTD57TuF1P2PQL+nZsLy3NcoA+D2JmuIeuJzt87c3S83FufRQraBKtstcEmhMSBPAp/bAOAB+gHFLYMLnHRKeRdsvk1ILxe+/GsdYa2vm7I8J+89k82ASEI4hXDwLZ5gT4HS1LBB9jgImT3307MNzK8DCa8AnQJrlP8Dpkz3lojBZbP2mQ6HoYGXONWhlmVY9Hj4puFGfFd6DDF1i+7LQHMvNDXIPyxlP4siMhdDjNoLCMJ/E0Ch+LWJJnYCGTqnqrBMDLwDBi4V3qD7hr0IfkHs6cDSZPnzZrs5KHv/wwOSoepStJEcuSCl4y4vR9uLvL/5mgNdvGR5DKk0mYw7pPsSnyXTBPL24TWERtKUBBVU/A8xkuRKMAellfERh/Odt3pbhpfNb7swkG/Rukd8j7nrfGyA0JjTPMWRQXq3ePdog1UV+XjSyByBXuSzW+Tyivw7eWCKX8gEmhlwB/Vqj9PhbKp2+rlEu/CX7YOPtrqQ5E/k88fpNHtPD9V+fIt/Pw3dDmI8/4z9QGw+k6K49azDkZ6BuFVicpZ4ZMZhGYEaKj9/7qi3b5AUFd8Ruli6If2sA0B2augJyaj0gqsQO3m4ToEJNoRwp/IT8vfI3Rk6PSm3kaT7whTmtls4EqZ0qX4ww5S7EWlwNjBTEW8mUIGBT7TAiZLhra0/e9yxvzzhftKetck1CU67/gusv9BIasq6n7OuMg/mJnJ0gw1m0R3KMChci3/eZFUxH6/xPcixMvPccj0BHctfSUeUhy/If601NlQH5pDAM/UDAXSwc4548TEAofRoDmhwBdsxUNt9Nwx/dh81yV6rI4FS+MW31IjN3mE4c6lCkXpBdQRrJrfmxkobcmmzk+mw19NPaHVIt7VCcpHg7IL3eJOrIuohzaDcgig9nM4dwz+IRcEKfT0yWNOD7qzBk25AeLJ1Q+QyXv/sXLJk7ngoqpw4PcUs+6GTmHN3AYU73WngRGlYQrTr7+LT44o+w51Dd9Gj8x2neFLqEOw+VpZAJMq1t8+b7xuG8c4luK34/I0JH32b+H5GeWuy+sjCzoWdrU1K/1Wljw1/8yjzkoScgmdCt5w3Ev/eh7EXEyrfSKlqsMb7yqCgEwyZRzP8Mwazb9nDYYzMS77YgGVdW+h6LzlQA2aVJs7GThN/27PqkWTsk/sbAJ/j4BSSVbcXTZf6LQ9+bteD0Zan2DAGMn1f5LQdmZxW2GOCA4Kn3OeEXPr9slJ4G9Ca9r2HEF/CSAfELSzfh3wzUduwcbI1UW5x/gYSWyuLL/c13cITGNUdv6KjxNYBCfjaegQzrULV10SXwF4abFdnpv0AARVLQOgpgaprIBSQgL+WNSOABlwsy1Hl2vrJefA3gW/HUTyOUv+HeX3ILOJ/ZXoUEOH0hFq7Wwlw3H4apdRHJPpT0EgZx2kt5NK+b4j81NIJAvuSFEN0gH65qjQ6lPl8r5zqc4yFUE1BVu+OiPhMbZ6R3HyZngRAgAd4k5skjHaXLO+6y3T9miWX96Mc0RV+/aI619JDpQtQf5TM6tGWjge+Rz3osOwlsuZuTTiGf0k+E0leYsSBvp6vncWfqKNB7piVQaM5uHtuzpl9WcVDq3pSspgBno/Rj+3qvVyM8tcy6+XmhH/LlR0mNOpr9sMOvWM6i8VRrsOPoxWO+crHyAujeGRcignfeb8tPFk8LxaQc2KObcBmd8Z7GrZULmdX5ASr7h1SJyPm698SA/WVYYAz1ldhmpCLpkOby1plXa4Z11sGNmnB9DTO0xCZ2TiOWnKoVJXgKtA0qto9utKPPUg/gokexpZ4FmfT+CBNoKkeoahRxFki+l85RGCXEWtsSNKeTOxIvdcXz2yGRSFG+rSxQAAAAAAAAAAA==",
```

Comment:
This should not happen. I'm investigating.

Reply:
Nice! Good to know that it shouldn't happen.